src/views/FarmAuction/components/AuctionLeaderboard/AuctionLeaderboardTable.tsx

```diff
@@ -0,0 +1,224 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import BigNumber from 'bignumber.js'
+import {
+  Text,
+  Flex,
+  Box,
+  OpenNewIcon,
+  BunnyPlaceholderIcon,
+  Skeleton,
+  Button,
+  useTooltip,
+  useMatchBreakpoints,
+  useModal,
+} from '@pancakeswap/uikit'
+import { getBscScanLink } from 'utils'
+import { getBalanceNumber } from 'utils/formatBalance'
+import EllipsisMenu, { OptionProps } from 'components/EllipsisMenu/EllipsisMenu'
+import { useTranslation } from 'contexts/Localization'
+import { usePriceCakeBusd } from 'state/farms/hooks'
+import { Auction, Bidder } from 'config/constants/types'
+import WhitelistedBiddersModal from '../WhitelistedBiddersModal'
+
+const LeaderboardContainer = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 5fr 3fr 1fr;
+  ${({ theme }) => theme.mediaQueries.md} {
+    grid-template-columns: 3fr 5fr 5fr 1fr;
+  }
+`
+
+const GridCell = styled(Flex)<{ isTopPosition }>`
+  height: 65px;
+  align-items: center;
+  border-bottom: 1px solid ${({ theme }) => theme.colors.cardBorder};
+
+  ${({ theme, isTopPosition }) =>
+    isTopPosition &&
+    `
```

Comment:
the formatting is messed up

Reply:
fixed in 2ca4cab6550efc4403a4658374e3005cf1bc12fc